bin/fasta_diff.py

```diff
@@ -281,6 +282,16 @@ def match_split_subsequence():
         if debug:
             fasta_dict_to_file(old_fasta_dict, old_fasta_file + '_stage_' + str(stage + 1) + '_unmatched')
             fasta_dict_to_file(new_fasta_dict, new_fasta_file + '_stage_' + str(stage + 1) + '_unmatched')
+        if report:
```

Comment:
Better practice is to use `if report is not None:` https://www.python.org/dev/peps/pep-0008/, but no difference here.